pkg/acsengine/k8s_versions.go

```diff
@@ -8,6 +8,33 @@ import (
 )
 
 var k8sComponentVersions = map[string]map[string]string{
+	"1.11": {
```

Comment:
@andyzhangx @feiskyer do you have any comments on k8s components compatibility with k8s 1.11?